.code-samples.meilisearch.yaml

```diff
@@ -175,7 +175,13 @@ update_settings_1: |-
         "synonyms": {
             "wolverine": ["xmen", "logan"],
             "logan": ["wolverine"]
-        }
+        },
+        "typoTolerance": {
+          "minWordSizeForTypos": {
+                "oneTypo": 8,
+                "twoTypos": 10
+      },
+            "disableOnAttributes": ["title"]
```

Comment:
It is cool to use a tool like this: https://jsonbeautifier.org/ before changing the samples :D

Reply:
Indeed, many thanks for the recommendation @brunoocasali 😄